Formula/ecl.rb

```diff
@@ -12,13 +12,24 @@ class Ecl < Formula
     sha256 "f86b66f182afdc3b7a8e8e51ca3e230d6ccc308d760faae1b2b345fd1ead0ab3" => :yosemite
   end
 
+  depends_on "pkg-config" => :run
   depends_on "gmp"
+  depends_on "bdw-gc"
+  depends_on "libffi"
+  depends_on "libatomic_ops" => :build
+  depends_on "autoconf" => :run
```

Comment:
Why is there a new dependency on `autoconf`? And why are the `pkg-config` and `autoconf` deps run-time?